main/glossary/README.md

```diff
@@ -39,6 +41,11 @@ Extents describe the extent of something that can be owned or shared: How much,
 
 Extent must be Comparable.
 
+## Handle
+A handle is a unique identifier implemented as a JavaScript object. Only its identity is meaningful, so most handles do not have properties. Unlike number or string identifiers, handles are unforgeable. This means no one who doesn't already have reference to the handle can create a fake a new reference to it.
+
+For example, Zoe contracts have an `offerHandle`. The `offerHandle` is the key for requesting the current allocation for this offer or reallocating the offerHandle's assets.
+
 ## Issuer
 Can create empty purses and payments, but it cannot mint new amounts. The issuer can also transform payments (splitting payments, combining payments, burning payments, and claiming payments exclusively). The issuer should be gotten from a trusted source and then relied upon as the decider of whether an untrusted payment is valid.
```

Comment:
## Issuer

Can create empty purses and payments, but cannot mint new amounts. Issuers can also transform payments (splitting, combining, burning, and claiming payments exclusively). You should get an issuer from a trusted source and then rely on it to determine if an untrusted payment is valid. Issuers are linked to a single mint and vice versa, so each issuer only works with one type of thing, such as only working with quatloos or only working with moola.

Reply:
Please check with someone to make sure that last is accurate. If it is, I think it's an important aspect of an issuer that should be mentioned in its definition.

Reply:
Your changes looks correct to me @tyg.

Reply:
I copied/pasted